### PR TITLE
Feat/Add:  jitsi_conference_sizes metric

### PR DESCRIPTION
**Pull request by RafPe**
Adds metrics for jitsi_conference_sizes

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -182,6 +182,12 @@ jitsi_total_packets_received_octo {{.TotalPacketsReceivedOcto}}
 # HELP jitsi_total_packets_sent_octo The total of sent dropped packets handled by the OCTO video bridge.
 # TYPE jitsi_total_packets_sent_octo gauge
 jitsi_total_packets_sent_octo {{.TotalPacketsSentOcto}}
+# HELP jitsi_conference_sizes Conference sizes
+# TYPE jitsi_conference_sizes gauge
+{{ range $key, $value := .ConferenceSizes -}}
+jitsi_conference_sizes{conferenceSize="{{$key}}"} {{ $value }}
```
Comment (0x46616c6b) [inline main.go:188]: Would suggest to use snake case for Prometheus metrics.

```suggestion
jitsi_conference_sizes{conference_size="{{$key}}"} {{ $value }}
```

File: main.go [hunk 2]
```diff
@@ -182,6 +182,12 @@ jitsi_total_packets_received_octo {{.TotalPacketsReceivedOcto}}
 # HELP jitsi_total_packets_sent_octo The total of sent dropped packets handled by the OCTO video bridge.
 # TYPE jitsi_total_packets_sent_octo gauge
 jitsi_total_packets_sent_octo {{.TotalPacketsSentOcto}}
+# HELP jitsi_conference_sizes Conference sizes
```
Comment (0x46616c6b) [inline main.go:185]:
```suggestion
# HELP jitsi_conference_sizes Distribution of conference sizes
```

File: main_test.go [hunk 3]
```diff
@@ -146,6 +146,30 @@ jitsi_total_packets_received_octo 0
...
+jitsi_conference_sizes{conferenceSize="12"} 0
+jitsi_conference_sizes{conferenceSize="13"} 0
+jitsi_conference_sizes{conferenceSize="14"} 0
+jitsi_conference_sizes{conferenceSize="15"} 0
+jitsi_conference_sizes{conferenceSize="16"} 0
+jitsi_conference_sizes{conferenceSize="17"} 0
+jitsi_conference_sizes{conferenceSize="18"} 0
+jitsi_conference_sizes{conferenceSize="19"} 0
+jitsi_conference_sizes{conferenceSize="20"} 0
+jitsi_conference_sizes{conferenceSize="21"} 0
```
Comment (0x46616c6b) [inline main_test.go:172]:
```suggestion
# HELP jitsi_conference_sizes Distribution of conference sizes
# TYPE jitsi_conference_sizes gauge
jitsi_conference_sizes{conference_size="0"} 0
jitsi_conference_sizes{conference_size="1"} 1
jitsi_conference_sizes{conference_size="2"} 0
jitsi_conference_sizes{conference_size="3"} 1
jitsi_conference_sizes{conference_size="4"} 0
jitsi_conference_sizes{conference_size="5"} 0
jitsi_conference_sizes{conference_size="6"} 0
jitsi_conference_sizes{conference_size="7"} 0
jitsi_conference_sizes{conference_size="8"} 0
jitsi_conference_sizes{conference_size="9"} 0
jitsi_conference_sizes{conference_size="10"} 0
jitsi_conference_sizes{conference_size="11"} 0
jitsi_conference_sizes{conference_size="12"} 0
jitsi_conference_sizes{conference_size="13"} 0
jitsi_conference_sizes{conference_size="14"} 0
jitsi_conference_sizes{conference_size="15"} 0
jitsi_conference_sizes{conference_size="16"} 0
jitsi_conference_sizes{conference_size="17"} 0
jitsi_conference_sizes{conference_size="18"} 0
jitsi_conference_sizes{conference_size="19"} 0
jitsi_conference_sizes{conference_size="20"} 0
jitsi_conference_sizes{conference_size="21"} 0
```

File: main_test.go [hunk 4]
```diff
@@ -333,6 +357,30 @@ jitsi_total_packets_received_octo 0
...
```
Comment (0x46616c6b) [inline main_test.go:383]:
```suggestion
# HELP jitsi_conference_sizes Distribution of conference sizes
# TYPE jitsi_conference_sizes gauge
jitsi_conference_sizes{conference_size="0"} 0
jitsi_conference_sizes{conference_size="1"} 1
jitsi_conference_sizes{conference_size="2"} 0
jitsi_conference_sizes{conference_size="3"} 1
jitsi_conference_sizes{conference_size="4"} 0
jitsi_conference_sizes{conference_size="5"} 0
jitsi_conference_sizes{conference_size="6"} 0
jitsi_conference_sizes{conference_size="7"} 0
jitsi_conference_sizes{conference_size="8"} 0
jitsi_conference_sizes{conference_size="9"} 0
jitsi_conference_sizes{conference_size="10"} 0
jitsi_conference_sizes{conference_size="11"} 0
jitsi_conference_sizes{conference_size="12"} 0
jitsi_conference_sizes{conference_size="13"} 0
jitsi_conference_sizes{conference_size="14"} 0
jitsi_conference_sizes{conference_size="15"} 0
jitsi_conference_sizes{conference_size="16"} 0
jitsi_conference_sizes{conference_size="17"} 0
jitsi_conference_sizes{conference_size="18"} 0
jitsi_conference_sizes{conference_size="19"} 0
jitsi_conference_sizes{conference_size="20"} 0
jitsi_conference_sizes{conference_size="21"} 0
```

**Review by 0x46616c6b — CHANGES_REQUESTED** [review]
Thanks for this contribution. I only have a minor change request regarding the nameing/labels.

**Comment by RafPe** [comment]
I have applied the suggestions :)

**Review by RafPe — COMMENTED** [review]
Implemented changes

**Review by 0x46616c6b — APPROVED** [review]

**Pull request merged**